### DISCUSSION TITLE
Hacktoberfest2024

### DISCUSSION THREAD

**Discussion by BShreyaa18** [TheAlgorithms/Python#12219] [Q&A]
Is this repo taking contributions for hacktoberfest this year?

**Comment by yashwanth-adimulam** [comment]
Yes it is.

**Comment by Muxumed** [comment]
hell yeah

**Comment by michaelabechynova** [comment]
> Is this repo taking contributions for hacktoberfest this year?

😜

**Comment by michaelabechynova** [comment]
> Is this repo taking contributions for hacktoberfest this year?
